fachhochschule-vorarlberg.csl

```diff
@@ -1,33 +1,75 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
   <info>
-    <title>Fachhochschule Vorarlberg (German)</title>
-    <title-short>FHV</title-short>
+    <title>Fachhochschule Vorarlberg Autor-Jahr</title>
+    <title-short>FHV Autor-Jahr</title-short>
     <id>http://www.zotero.org/styles/fachhochschule-vorarlberg</id>
-    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg" rel="self"/>
+    <link href="http://www.zotero.org/styles/fh-vorarlberg-autor-jahr" rel="self"/>
```

Comment:
Hallo Herr Karcher,

Herzlichen Dank für Ihre Feedbacks. Leider bin ich bzw. sind wir keine Programmierer, sodass die Umsetzung unseres Zitierstiles auf mühsamer Trial-and-Error-Basis erfolgt. Trotzdem konnten wir einige Bugs des Zitierstils beheben und versuchen nun, die aktualisierte Fassung über die Zotero-Website zugänglich zu machen.
ich habe Ihre Anweisungen befolgt und die geänderte Version des Stils validieren lassen - mit positivem Ergebnis. Bezüglich des Titels sind aber noch einige Fragen offen, weshalb ich mit dem Pull Request noch auf Ihre Antworten warte.

Viele Grüsse,
Frank Weber


On 07.03.2016, at 14:50, Sebastian Karcher wrote:

> In fachhochschule-vorarlberg.csl:
> 
> >      <id>http://www.zotero.org/styles/fachhochschule-vorarlberg</id>
> > -    <link href="http://www.zotero.org/styles/fachhochschule-vorarlberg" rel="self"/>
> > +    <link href="http://www.zotero.org/styles/fh-vorarlberg-autor-jahr" rel="self"/>
> this here is your problem. Revert that change back, please.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
erledigt.

- - - - -
In fachhochschule-vorarlberg.csl:

> @@ -1,33 +1,75 @@
>  <?xml version="1.0" encoding="utf-8"?>
>  <style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="de-DE">
>    <info>
> -    <title>Fachhochschule Vorarlberg (German)</title>
> -    <title-short>FHV</title-short>
> +    <title>Fachhochschule Vorarlberg Autor-Jahr</title>
The (German) still needs to be there, please. And is there a note style, or why do you need the "Autor-Jahr"?

Wir planen, eine Fussnoten-Version unseres Stiles zur Verfügung zu stellen, deswegen habe ich das Autor-Jahr in den Titel hineingenommen. Aber: Ist es überhaupt möglich, für eine Institution 2 Stile im Zotero Stile Repository zu haben? Falls ja, ist "Fachhochschule Vorarlberg Autor-Jahr (German)" als Titel zulässig?
Gibt es irgendeine Möglichkeit, eine Versionierungs-Information im Titel unterzubringen, also z.B. "Version 2.1" oder dergleichen?
- - - - - - 

> In fachhochschule-vorarlberg.csl:
> 
> > @@ -101,7 +145,8 @@
> >        <else-if type="article-journal article-newspaper article-magazine entry-encyclopedia" match="any">
> >          <choose>
> >            <if variable="author">
> > -            <text variable="title" font-style="normal" prefix="„" suffix="“"/>
> > +            <!--suffix um . vor dem Hochkomma erweitert! lt. kuma (zeitschriftenartikel)-->
> > +            <text variable="title" font-style="normal" prefix="„" suffix=".“"/>
> is there any reason we're not doing these with quotes="true" instead of the hardcocded quotation marks?
> 
> 
Incompetence, I'm afraid

> In fachhochschule-vorarlberg.csl:
> 
> > @@ -109,7 +154,8 @@
> >              </choose>
> >            </if>
> >            <else>
> > -            <text variable="title" font-style="normal" prefix="„" suffix="“"/>
> > +            <!--suffix um . vor dem Hochkomma erweitert! lt. kuma -->
> > +            <text variable="title" font-style="normal" prefix="„" suffix=".“"/>
> ditto.
> 
> 
See above.

> In fachhochschule-vorarlberg.csl:
> 
> >        <text variable="locator" prefix=",&#160;S.&#160;"/>
> >      </layout>
> >    </citation>
> >    <!-- Hier beginnt die Formatierung für das Literaturverzeichnis -  this is where the bibliography starts -->
> > -  <bibliography hanging-indent="true" et-al-min="4" et-al-use-first="1">
> > +  <!-- normale Version -->
> > +  <!--<bibliography hanging-indent="true" et-al-min="4" et-al-use-first="1">-->
> Wenn das die "normale Version" ist, sollte die dann nicht im Stil bleiben?
> 
> 
> 
Ja, danke für den Hinweis.